### ISSUE TITLE
IdsToolbarSection: align attribute not doing anything

### ISSUE THREAD

**Issue by whernebrink** [infor-design/enterprise-wc#931]
**Describe the bug**
Setting e.g. `align="end"` on the `ids-toolbar-section` does not work. Doing (in angular) with property binding `[align]="'end'"` does work.

**To Reproduce**
1. Go to https://stackblitz.com/edit/angular-ivy-osnfi9?file=src/app/app.component.html
2. See on line 21 that there is currently the property binding, which work.
3. Change to just an attribute `align="end"`
4. See toolbar section not aligned correctly, which you probably could tell by looking the align attribute, which now is  red in stackblitz (which must mean that there is something up with it 🤷‍♂️).

**Expected behavior**
Be able to align via the align attribute. 

**Version**

- ids-enterprise-wc: 1.0.0-beta.1

**Screenshots**
![image](https://user-images.githubusercontent.com/73601152/190713353-ef649402-a489-44ae-9697-c814931a195d.png)

**Platform**
 - Infor Application/Team Name: M3
 - OS Version: macOS Monteray
 - Browser Name: Firefox 
 - Browser Version: 104.0.2

**Comment by EdwardCoyle** [comment]
@whernebrink I played around with this example a bit and realized that there were some imports and "page-level" HTML markup missing (mostly the part that imports fonts).  I also noticed there were still remnants of "Text Button 1" that seemed to get in the way of the left-side alignment of the items.

I was able to fix these things ([See this updated sample](https://stackblitz.com/edit/angular-ivy-kfrydh?file=src/app/app.component.html)), and set the `align="start"` property to get the expected result in the Toolbar section.  I can't speak for why the editor is showing that property as if it's an error, but I can see the alignment working properly and it does change if you make it "center" or "end".

For good measure I also added some examples of the Toolbar to our Angular example repo in infor-design/enterprise-wc-examples#17 that also seemed to be working against our latest main branch.
